lib/yt/models/video.rb

```diff
@@ -247,6 +247,12 @@ def licensed?
         content_detail.licensed_content || false
       end
 
+      # @return [Boolean] whether the video is identified by YouTube as
+      #   age-restricted content.
+      def age_restricted?
+        content_detail.content_rating != nil
```

Comment:
Looking back at your https://github.com/Fullscreen/yt/issues/173#issuecomment-104075497 I realize that maybe what you intended to go for is to check if there is any value for `contentDetails.contentRating.ytRating` instead.